### PR TITLE
Cuda9.1 upgrade

### PR DESCRIPTION
**Pull request by millerhooks**

### CODE REVIEW

File: container-images/js-and-gpu/Dockerfile [hunk 1]
```diff
@@ -1,30 +1,10 @@
...
-    gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key"; \
-  done
+FROM graphistry/gpu-base:9.1
 
-ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 9.6.0
+RUN apt-get update && apt-get install -y \
+        build-essential \
+        curl && \
+    curl -sL http://deb.nodesource.com/setup_9.x | bash && \
```
Comment (lsb) [inline container-images/js-and-gpu/Dockerfile]: We're curling something fetched not via TLS and piping it into bash?

Comment (lsb) [reply container-images/js-and-gpu/Dockerfile]: This is based on how the Node dockerfile is created btw https://github.com/nodejs/docker-node/blob/master/9/Dockerfile

Comment (trxcllnt) [reply container-images/js-and-gpu/Dockerfile]: nodesource hosts on https too: https://deb.nodesource.com/setup_9.x

Comment (trxcllnt) [reply container-images/js-and-gpu/Dockerfile]: also 10: https://deb.nodesource.com/setup_10.x

**Review by trxcllnt — APPROVED** [review]

**Pull request merged**